website/src/styles.scss

```diff
@@ -0,0 +1,457 @@
+$font-stack:    Helvetica, sans-serif;
```

Comment:
Most of this is like copy-pasted from somewhere leading to lots of unnecessary rules. Also, it tries to style all components used in the app. Components should contain styles relevant to them, this file is for global styles - setting up body, application-wide typography, etc.